src/js/components/form-elements/FilterInput.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+
+import PropTypes from 'prop-types';
+import { getTranslate } from 'react-localize-redux';
+import { connect } from 'react-redux';
+
+import { translateWithDefaultMessage } from 'utils/Translate';
+
+const FilterInput = ({
+  itemFilter, onChangeFunc, clearFunc, translate,
```

Comment:
```suggestion
  itemFilter, onChange, onClear, translate,
```
onClick, onClear etc.. is a more conventional name for listeners in React.